src/client/Microsoft.Identity.Client/Instance/Validation/AdfsAuthorityValidator.cs

```diff
@@ -36,6 +36,7 @@ public AdfsAuthorityValidator(RequestContext requestContext)
 
                 if (httpResponse.StatusCode != HttpStatusCode.OK)
                 {
+                    _requestContext.Logger.Error($"Authority validation failed due to invalid authority. Authority: {authorityInfo.CanonicalAuthority}");
```

Comment:
```suggestion
                    _requestContext.Logger.Error($"Authority validation failed because the configured authority is invalid. Authority: {authorityInfo.CanonicalAuthority}");
```

Reply:
nit